src/EnergyPlus/WaterThermalTanks.hh

```diff
@@ -768,7 +768,13 @@ namespace WaterThermalTanks {
     void CalcWaterThermalTankStratified(int const WaterThermalTankNum // Water Heater being simulated
     );
 
-    void CalcNodeMassFlows(int const WaterThermalTankNum, // Water Heater being simulated
+    void CalcWaterThermalTankStratifiedOld(int const WaterThermalTankNum // Water Heater being simulated
+    );
+
```

Comment:
Is the old calc just here for testing and will eventually be removed completely?

Reply:
Yes, that's the plan. It's made a nice way to switch between the two as I developed since this is more or less a rewrite of the old model and I wanted to compare results. At one point I thought maybe we could leave both in and let people choose which model they wanted with a flag in the idf object, but eventually talked myself out of it.